userguide/platform/includes/killbill-configuration-properties-table.adoc

```diff
@@ -1075,3 +1075,13 @@ KB_org_killbill_manualPayTemplate_name   |Path to the invoice template for accou
 KB_org_killbill_template_invoiceFormatterFactoryPluginName   |Invoice formatter factory plugin name   |null   |Config File/Environment Variable
 |===
 
+=== Queue Properties
+
+[options="header",cols="1,1,1,1"]
+|===
+|Property Name/Environment Variable Name   |Description   |Default Value | Configuration Method
+//-------------------------------------------------
+|org.killbill.queue.creator.name   |Allow to override the default naming based on Hostname  |-  |System Property
```

Comment:
In Kill Bill, these can indeed only be set using System Properties. However, with Docker, it is possible to configure these via environment properties. I think it would be worth mentioning it (it's a special case):

* `KILLBILL_QUEUE_CREATOR_AUTOGENERATE_NAME` (true by default)
* `KILLBILL_QUEUE_CREATOR_NAME` empty by default